checkstyle/import-control.xml

```diff
@@ -20,6 +20,7 @@
     <allow pkg="com.google"/>
     <allow pkg="io.pravega"/>
     <allow pkg="lombok"/>
+    <allow pkg="edu.umd.cs.findbugs.annotations"/>
```

Comment:
Yes. I have a local change that will remove both the need for the import and the warning being generated in the first place. It changes the lombok config to work with findbugs so that generated code is not scanned. I didn't combine it with this PR to keep the concerns separate. I will send a PR for that change immediately after this goes in, and it will remove this line.